test/rendition.postprocess.test.js

```diff
@@ -190,11 +193,19 @@ describe("imagePostProcess", () => {
 
         // check metrics
         await MetricsTestHelper.metricsDone();
+        console.log('receivedMetrics---->',receivedMetrics);
         assert.equal(receivedMetrics[0].eventType, "rendition");
         assert.equal(receivedMetrics[0].callbackProcessingDuration + receivedMetrics[0].postProcessingDuration, receivedMetrics[0].processingDuration);
         assert.equal(receivedMetrics[3].eventType, "activation");
         assert.equal(receivedMetrics[3].callbackProcessingDuration, receivedMetrics[0].callbackProcessingDuration, receivedMetrics[1].callbackProcessingDuration, receivedMetrics[2].callbackProcessingDuration);
         assert.equal(receivedMetrics[3].callbackProcessingDuration + receivedMetrics[3].postProcessingDuration, receivedMetrics[3].processingDuration);
+        // downloadDuration: 0.002481363,
+        // callbackProcessingDuration: 0.002246784,
+        // postProcessingDuration_id_mean: 449890804.5,
+        // postProcessingDuration_hrtimeElapsed_mean: 1115159,
+        // processingDuration: '0.002246784[object Object]',
+        // uploadDuration: 0.057330613999999995
```

Comment:
remove comment

Reply:
I would do fuzzy checks - I didn't see time being mocked and we can't exclude small variations. Or did I miss something in the code?